src/app/workflow/workflow.component.ts

```diff
@@ -194,6 +196,10 @@ export class WorkflowComponent extends Entry {
           this.setPublishMessage();
           this.selectedVersion = this.selectVersion(this.workflow.workflowVersions, this.urlVersion,
             this.workflow.defaultVersion);
+          if (this.selectedVersion) {
```

Comment:
Just for my own understanding, the workflow component updates the Ga4gh Files entity store on initialization and each time a different workflow version is selected by the user?

Reply:
yes